source/rust_verify/example/invariants.rs

```diff
@@ -1,46 +1,46 @@
-#[allow(unused_imports)]
```

Comment:
This file appears (mostly?) unchanged but GitHub shows it as completely rewritten. Are these whitespace changes? Could you maintain the existing formatting, so we can keep track of exactly what change was made as result of this?

Reply:
> This file appears (mostly?) unchanged but GitHub shows it as completely rewritten. Are these whitespace changes (or perhaps a line terminator changes)? Could you maintain the existing formatting, so we can keep track of exactly what change was made as result of this?

Fixed.